### PR TITLE
Tofe/qml work

### PR DESCRIPTION
**Pull request by Tofee**

### CODE REVIEW

File: data/appinfo.about.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Device Info",
+    "id": "org.webosports.app.settings.about",
+    "version": "0.4.0",
+	"vendor": "WebOS Ports",
```
Comment (Herrie82) [inline data/appinfo.about.json]: Can you cleanup the whitespaces here? Seems some are tab and some space. Also lower.

File: src/qml/General/AboutPage.qml [hunk 2]
```diff
@@ -16,9 +16,169 @@
...
+// Units & font sizes
+import LunaNext.Common 0.1
 
 import "../Common"
 
 BasePage {
 
+    property alias deviceName:        deviceNameLabel.value
+    property alias deviceSerial:      deviceSerialLabel.value
+    property alias deviceWifiAddress: deviceWifiAddrLabel.value
```
Comment (Herrie82) [inline src/qml/General/AboutPage.qml]: Same here not aligned with items below

Comment (Tofee) [reply src/qml/General/AboutPage.qml]: I wanted to do 2 groups here; I find that having too many things aligned is actually harder to read sometimes.

Comment (Herrie82) [reply src/qml/General/AboutPage.qml]: OK

**Pull request merged**